src/commands/actor/get-public-url.ts

```diff
@@ -27,6 +29,20 @@ export class GetPublicUrlCommand extends ApifyCommand<typeof GetPublicUrlCommand
 		const apiBase = process.env[APIFY_ENV_VARS.API_PUBLIC_BASE_URL];
 		const storeId = process.env[ACTOR_ENV_VARS.DEFAULT_KEY_VALUE_STORE_ID];
 
-		console.log(`${apiBase}/v2/key-value-stores/${storeId}/records/${key}`);
+		const apifyClient = await getApifyStorageClient();
+		const store = await apifyClient.keyValueStore(storeId!).get();
+
+		const publicUrl = new URL(`${apiBase}/v2/key-value-stores/${storeId}/records/${key}`);
+
+		if (store) {
+			// @ts-expect-error Add types to client
```

Comment:
Isn't expect-error a bit too much? That way, typescript will complain if the type of `store` gets extended in the future, won't it?

Reply:
It's not, it should be used to mark something that will be solved in the future with a typing update! When we fix the types, we just have to drop this comment